bazel/protobuf.bzl

```diff
@@ -75,7 +75,7 @@ def _protoc_python_output_files(proto_file_sources):
     for p in proto_file_sources:
         basename = p.basename[:-len(".proto")]
 
-        python_srcs.append(basename + "_pb2.py")
+        python_srcs.append(basename.replace("-", "_", maxsplit = None) + "_pb2.py")
```

Comment:
@akonradi yeah, I'm not sure about this one though.

Reply:
This seems fine. It shouldn't break anyone since they would already be broken by filenames with hyphens; we just didn't know about it.